ObjectiveGit/GTRepository.h

```diff
@@ -565,6 +565,20 @@ NS_ASSUME_NONNULL_BEGIN
 /// Returns whether `ahead` and `behind` were successfully calculated.
 - (BOOL)calculateAhead:(size_t *)ahead behind:(size_t *)behind ofOID:(GTOID *)headOID relativeToOID:(GTOID *)baseOID error:(NSError **)error;
 
+/// Creates an enumerator for walking the local commits in relative to the
+/// tracking branch.
+///
+/// Note that this will only be accurate iff `trackingBranch` truly is
+/// `localBranch`s tracking branch. If you need more general functionality, see
+/// -[GTBranch uniqueCommitsRelativeToBranch:error:].
+///
+/// localBranch    - The local branch.
+/// trackingBranch - `localBranch`s tracking branch.
+/// error          - The error if one occurred.
+///
+/// Returns the enumerator or nil if an error occurred.
+- (nullable GTEnumerator *)enumerateLocalCommitsInBranch:(GTBranch *)localBranch trackingBranch:(GTBranch *)trackingBranch error:(NSError **)error;
```

Comment:
We're using this terminology elsewhere, e.g., `enumerateUniqueCommitsUpToOID`. I'm not opposed to using something else here, but it's also not without precedent.